app/config.go

```diff
@@ -196,43 +193,41 @@ func loadExternalProperties(properties []*data.Attribute) (map[string]interface{
 		}
 	}
 
+	if len(props) > 0 {
+		// Get value using overridden property name
+		for k, v := range props {
```

Comment:
Should we remove this code? I didn't see any value.

Reply:
@lixingwang: It does in case only `FLOGO_APP_PROPS_OVERRIDE` is set.